package/libwebsockets/Config.in

```diff
@@ -9,5 +9,13 @@ config BR2_PACKAGE_LIBWEBSOCKETS
 
 	  https://libwebsockets.org/
 
+config BR2_PACKAGE_LIBWEBSOCKETS_NETLINK
+	bool "enable netlink monitoring"
+	default y
+	help
+	  libwebsockets will track link status with netlink
+	  on linux if avilable. turn off to prevent potential
+	  issues with IPv6 support.
```

Comment:
nit:  I couldn't find an explicit rule that the help must be grammatically correct, but the examples and other packages are so should s/turn/Turn/ if this is going upstream.

Reply:
Fair point, will fix my terrible grammar.